docs/resources/grant_privileges_to_share.md

```diff
@@ -0,0 +1,108 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "snowflake_grant_privileges_to_share Resource - terraform-provider-snowflake"
+subcategory: ""
+description: |-
+  
+---
+
+# snowflake_grant_privileges_to_share (Resource)
+
+
+
+## Example Usage
+
+```terraform
+##################################
+### on database
+##################################
+
+resource "snowflake_grant_privileges_to_share" "example" {
+  share_name  = snowflake_share.example.name
+  privileges  = ["USAGE"]
+  on_database = snowflake_database.example.name
```

Comment:
No other resource was designed with `on { ... }` block, it is an interesting idea, though. We should have discussed that during the design, I am not sure if we want to open this can of worms again

Reply:
changed to "to_share" and for others like this "database_name" -> "on_database"